modules/apps/data-engine/data-engine-test/src/testFunctional/macros/DERenderer.macro

```diff
@@ -465,15 +465,4 @@ definition {
 			locator1 = "DataEngineRenderer#REPEATABLE_FIELD_DELETE_BUTTON_INDEXED");
 	}
 
-	macro assertDataInUploadField {
-		if (!(isSet(index))) {
-			var index = "1";
-		}
-
-		AssertTextEquals(
-			key_fieldLabel = "${fieldLabel}",
-			locator1 = "TextInput#SELECTOR_INPUT",
-			value1 = "${webContentUpload}");
-	}
-
```

Comment:
@guivilarim, the commit message is inconsistent with the commit content, can you please revisit this commit?